.gitignore

```diff
@@ -84,10 +82,6 @@ local.properties
 .idea/artifacts
 .idea/compiler.xml
 .idea/jarRepositories.xml
-.idea/modules.xml
-.idea/*.iml
-.idea/modules
-*.iml
```

Comment:
Why are all of these removed? I tested this change and now have these untracked files:
```
	.idea/deploymentTargetDropDown.xml
	.idea/inspectionProfiles/
```